spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAncestorsOfProcedure.java

```diff
@@ -156,7 +156,7 @@ public void testInvalidAncestorOfCases() {
     AssertHelpers.assertThrows(
         "Should reject calls with empty table identifier",
         IllegalArgumentException.class,
-        "Cannot handle an empty identifier for argument table",
+        "Cannot handle an empty identifier for parameter 'table'",
```

Comment:
Does the exception message need to change?

Reply:
We reworked error messages in `ProcedureInput` and since this procedure now consumes it, I had to update the test.